hack/local-up-cluster.sh

```diff
@@ -131,6 +131,10 @@ sudo "${GO_OUT}/kube-scheduler" \
   --master="http://${API_HOST}:${API_PORT}" >"${SCHEDULER_LOG}" 2>&1 &
 SCHEDULER_PID=$!
 
```

Comment:
Done. It now echos out the commands to run to set kubeconfig entries to talk to the local cluster.